mwdb/model/api_key.py

```diff
@@ -29,12 +28,9 @@ def issuer_login(self):
 
     @staticmethod
     def verify_token(token):
```

Comment:
We should handle legacy API keys as well. Scope verification will be breaking here. Audience verification may also fail.

API keys are the only JWT tokens that can't be regenerated easily, so we should keep them backwards-compatible.

I think it should be covered by additional test, I can help in preparing that one. We may need to set predictable secret for E2E testing to be able to hardcode legacy keys.